docs/explanation/ssh.md

```diff
@@ -0,0 +1,111 @@
+(juju-ssh)=
+# Juju SSH
+
```

Comment:
Wondering about relation to / implications for this other doc: https://documentation.ubuntu.com/juju/3.6/reference/ssh-key/